panel/src/components/Blocks/Types/Video.vue

```diff
@@ -9,7 +9,7 @@
     @update="update"
   >
     <k-aspect-ratio ratio="16/9">
-      <iframe v-if="video" :src="video" />
+      <iframe v-if="video" :src="video" referrerpolicy="strict-origin-when-cross-origin" />
```

Comment:
```suggestion
      <iframe v-if="video" :src="video" referrerpolicy />
```

`strict-origin-when-cross-origin` value is already default for `referrerpolicy`:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy#directives